src/server/completion.odin

```diff
@@ -790,114 +844,112 @@ get_implicit_completion :: proc(
 
 	//infer bitset and enums based on the identifier comp_lit, i.e. a := My_Struct { my_ident = . } 
 	if position_context.comp_lit != nil {
-		if position_context.parent_comp_lit.type == nil {
```

Comment:
This whole hunk/part of the diff is just turning this early return if into a regular if (in order to get to the part where call expressions are completed later). All this is just an indentation change.